lib/library.h

```diff
@@ -565,6 +565,8 @@ class CPPCHECKLIB Library {
     }
 };
 
+const Library::Container * getLibraryContainer(const Token * tok);
```

Comment:
```suggestion
CPPCHECKLIB const Library::Container * getLibraryContainer(const Token * tok);
```
Prepending `CPPCHECKLIB` works for me.